superset/db_engine_specs/base.py

```diff
@@ -161,6 +161,7 @@ class BaseEngineSpec:  # pylint: disable=too-many-public-methods
         utils.DbColumnType.STRING: (
             re.compile(r".*CHAR.*", re.IGNORECASE),
             re.compile(r".*STRING.*", re.IGNORECASE),
+            re.compile(r".*TEXT.*", re.IGNORECASE),
```

Comment:
This is mostly unrelated to this PR, but I noticed that the `TEXT` types were not being identified as string column types (I'm surprised this hadn't come up previously).